desktop/apps/auction2/filter_query.js

```diff
@@ -0,0 +1,67 @@
+export const filterQuery = `
```

Comment:
This replicates some of what's done in `/collect` already, but the whole impetus for this UI change is that we're going to be indexing on sale artwork for this page... meaning this query will change completely anyways.